boards/crazyflie/src/stm32-board.adb

```diff
@@ -173,4 +175,39 @@ package body STM32.Board is
       end if;
    end Configure_I2C;
 
+   procedure Initialize_EXT_SPI
```

Comment:
We add comment boxes above sub-program implementations:
```
------------------------
-- Initialize_EXT_SPI --
------------------------
```
You add a shortcut for that in GNATstudio, I use CTRL+B :)

Reply:
Done. Of course I can’t use GNATstudio, being on a Mac :-)

Reply:
As you said "The GPS from CE 2019 works just fine" :)